src/Latte/Runtime/Filters.php

```diff
@@ -316,26 +316,33 @@ public static function strip(FilterInfo $info, string $s): string
 	 * @param  bool  $strip  stripping mode
 	 * @return string HTML
 	 */
-	public static function spacelessHtml(string $s, int $phase = null, bool &$strip = true): string
+	public static function spacelessHtml(string $s, ?int $phase = null, bool &$strip = true): string
 	{
 		if ($phase & PHP_OUTPUT_HANDLER_START) {
 			$s = ltrim($s);
 		}
 		if ($phase & PHP_OUTPUT_HANDLER_FINAL) {
 			$s = rtrim($s);
 		}
-		return preg_replace_callback(
+		$return = (string) preg_replace_callback( // Other cases
 			'#[ \t\r\n]+|<(/)?(textarea|pre|script)(?=\W)#si',
-			function ($m) use (&$strip) {
+			static function (array $m) use (&$strip): string {
```

Comment:
Why `static`?

Reply:
PHP does not have to create a new instance every time.